akka-http-core/src/test/scala/akka/http/impl/engine/rendering/ResponseRendererSpec.scala

```diff
@@ -582,10 +582,13 @@ class ResponseRendererSpec extends FreeSpec with Matchers with BeforeAndAfterAll
     def awaitAtMost: FiniteDuration = 3.seconds
 
     def renderTo(expected: String): Matcher[HttpResponse] =
-      renderTo(expected, close = false) compose (ResponseRenderingContext(_))
+      renderToImpl(expected, checkClose = None) compose (ResponseRenderingContext(_))
 
     def renderTo(expected: String, close: Boolean): Matcher[ResponseRenderingContext] =
-      equal(expected.stripMarginWithNewline("\r\n") -> close).matcher[(String, Boolean)] compose { ctx ⇒
+      renderToImpl(expected, checkClose = Some(close))
+
+    def renderToImpl(expected: String, checkClose: Option[Boolean]): Matcher[ResponseRenderingContext] =
```

Comment:
I don't think I mind the manual delegation here, it's just a spec :) Thanks for the comment!